demo/demo.md

```diff
@@ -28,58 +28,103 @@ To meet our accessibility requirement, all uses of `auro-dropdown` should have a
 1. Use `aria-label` to insert label content that will only be read by screen readers
 1. Use `aria-labeledby` to append a description from another element on the page
 
-Not including one of the above options will result in your UI being non-compliant with Alaska's accessibility policies.
-
+Not including one of the above options will result in your UI being non-compliant with Alaska's accessibility policies.
+
 ## Common use with auro-label
 
 This first common example uses the default `auro-dropdown` element with the attributes of `bordered` `rounded` `inset` `toggle` and `chevron`. Additionally the `aria-label` attribute is used to define a string value that labels an interactive element.
 
-<div class="exampleWrapper">
-  <auro-dropdown id="common" common aria-label="Label content for screen reader">
+
+### modal
+
+<div class="exampleWrapper">
+  <auro-dropdown id="common" common modal="sm" aria-label="Label content for screen reader">
     <div style="padding: var(--auro-size-sm);">
       Lorem ipsum solar
       <br />
+      <input />
       <auro-button onclick="document.querySelector('#common').hide()">
         Dismiss Dropdown
       </auro-button>
+      <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Purus in mollis nunc sed. Cursus eget nunc scelerisque viverra. Ligula ullamcorper malesuada proin libero nunc consequat interdum. Viverra vitae congue eu consequat ac felis. Amet volutpat consequat mauris nunc congue nisi. Praesent tristique magna sit amet purus gravida quis blandit turpis. Risus sed vulputate odio ut enim blandit volutpat maecenas volutpat. Consectetur lorem donec massa sapien faucibus et molestie. Non odio euismod lacinia at quis risus sed vulputate. Id ornare arcu odio ut sem. Eget mi proin sed libero. Dui accumsan sit amet nulla. Tellus integer feugiat scelerisque varius morbi enim. Fermentum iaculis eu non diam phasellus vestibulum. Sed risus pretium quam vulputate dignissim suspendisse in est ante. Dolor morbi non arcu risus quis varius.</p>
+      <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Purus in mollis nunc sed. Cursus eget nunc scelerisque viverra. Ligula ullamcorper malesuada proin libero nunc consequat interdum. Viverra vitae congue eu consequat ac felis. Amet volutpat consequat mauris nunc congue nisi. Praesent tristique magna sit amet purus gravida quis blandit turpis. Risus sed vulputate odio ut enim blandit volutpat maecenas volutpat. Consectetur lorem donec massa sapien faucibus et molestie. Non odio euismod lacinia at quis risus sed vulputate. Id ornare arcu odio ut sem. Eget mi proin sed libero. Dui accumsan sit amet nulla. Tellus integer feugiat scelerisque varius morbi enim. Fermentum iaculis eu non diam phasellus vestibulum. Sed risus pretium quam vulputate dignissim suspendisse in est ante. Dolor morbi non arcu risus quis varius.</p>
```

Comment:
I am confused by this demo. Clicking on the trigger doesn't open a modal, but opens a dropdown that is 100% across? Is that intentional? 

Also, there are no code examples for the demo.